### PR TITLE
Revert "files to be reviewed"

### PR DESCRIPTION
**Pull request by mlburggr**
Reverts orangerbt/GPSDataAnalysis#8

### CODE REVIEW

File: analysis.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    '''
+
+    def gprmcParse(self, gprmc, line):
+        gprmc[1] = line[1][0:2] + ":" + line[1][2:4] + ":" + line[1][4:6]
+        gprmc[2] = "OK" if line[2] == "A" else "WARNING"
+
+        gprmc[3] = [line[3][-4:]]
+        line[3] = line[3][-4:]
+        gprmc[3].append(line[3])
+        gprmc[3].append(line[4])
```
Comment (gjsmo) [inline analysis.py:41]: Change from degrees, minutes and bearing (N or S) to decimal degrees and + for north or - for south.

File: analysis.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    gprmc[8] - Magnetic variation, format: [degrees, cardinal direction]
+    gprmc[9] - Checksum
+    '''
+
+    def gprmcParse(self, gprmc, line):
+        gprmc[1] = line[1][0:2] + ":" + line[1][2:4] + ":" + line[1][4:6]
+        gprmc[2] = "OK" if line[2] == "A" else "WARNING"
+
+        gprmc[3] = [line[3][-4:]]
+        line[3] = line[3][-4:]
```
Comment (gjsmo) [inline analysis.py:39]: Redundant?

File: analysis.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        gprmc[3].append(line[3])
+        gprmc[3].append(line[4])
+
+        gprmc[4] = [line[5][-4:]]
+        line[5] = line[5][-4:]
+        gprmc[4].append(line[5])
+        gprmc[4].append(line[6])
```
Comment (gjsmo) [inline analysis.py:46]: Change from degrees, minutes and bearing (E or W) to decimal degrees and + for east or - for west

File: analysis.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        gprmc[1] = line[1][0:2] + ":" + line[1][2:4] + ":" + line[1][4:6]
+        gprmc[2] = "OK" if line[2] == "A" else "WARNING"
+
+        gprmc[3] = [line[3][-4:]]
+        line[3] = line[3][-4:]
+        gprmc[3].append(line[3])
+        gprmc[3].append(line[4])
+
+        gprmc[4] = [line[5][-4:]]
+        line[5] = line[5][-4:]
```
Comment (gjsmo) [inline analysis.py:44]: Redundant?

File: analysis.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    '''
+    GPRMC properties
+
+    gprmc[0] - "Recommended minimum specific GPS/Transit data"
+    gprmc[1] - Time of fix, UTC
+    gprmc[2] - Navigation receiver warning (either OK or WARNING)
+    gprmc[3] - Latitude, format: [degrees, minutes, cardinal direction]
+    gprmc[4] - Longitude, format: [degrees, minutes, cardinal directon]
+    gprmc[5] - Speed over ground (in knots)
+    gprmc[6] - Course Made Good
```
Comment (gjsmo) [inline analysis.py:28]: Just "Course (in degrees)"

File: analysis.py [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        gprmc[3].append(line[4])
+
+        gprmc[4] = [line[5][-4:]]
+        line[5] = line[5][-4:]
+        gprmc[4].append(line[5])
+        gprmc[4].append(line[6])
+
+        gprmc[5] = line[7]
+        gprmc[6] = line[8]
+        gprmc[7] = line[9][4:6] + "/" + line[9][0:2] + "/" + line[9][2:4]
```
Comment (gjsmo) [inline analysis.py:50]: ISO format (yyyy-mm-dd)

File: analysis.py [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+        gprmc[8] = [line[10], line[11]]
+        gprmc[9] = line[12]
```
Comment (gjsmo) [inline analysis.py:52]: Don't output checksum - but check if it's valid

**Review by gjsmo — APPROVED** [review]

File: analysis.py (hunk 4 above)
Comment (mlburggr) [reply analysis.py:44]: Fixed

File: analysis.py (hunk 5 above)
Comment (mlburggr) [inline analysis.py]: Fixed

File: analysis.py (hunk 2 above)
Comment (mlburggr) [reply analysis.py:39]: Fixed

File: analysis.py (hunk 5 above)
Comment (mlburggr) [reply analysis.py:28]: Fixed

File: analysis.py (hunk 1 above)
Comment (mlburggr) [reply analysis.py:41]: Done

File: analysis.py (hunk 3 above)
Comment (mlburggr) [reply analysis.py:46]: done

File: analysis.py (hunk 6 above)
Comment (mlburggr) [reply analysis.py:50]: done

File: analysis.py (hunk 7 above)
Comment (mlburggr) [reply analysis.py:52]: Will do this in a later revision, made issue #10

**Pull request merged**